src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -397,11 +397,11 @@ class RecipeMarkdownGenerator : Runnable {
                 to align the versions of Rewrite's modules to ensure compatibility.
                 The use of the "bill of materials" means that a developer will only need to specify explicit versions of the BOM and the build plugins:
 
```

Comment:
Do we want to include [these lines](https://github.com/openrewrite/rewrite-docs/blob/ff4853be43cd3a3930fca2eecd307413d30df1f9/docs/licensing/repository-licensing.md#L3-L9) here as well?
```suggestion

The three licenses in the table can be found here:

* [Apache License Version 2.0](https://www.apache.org/licenses/LICENSE-2.0)
* [Moderne Source Available License](https://docs.moderne.io/licensing/moderne-source-available-license)
* [Moderne Proprietary License](https://docs.moderne.io/licensing/overview)

```

Reply:
I feel like we either need to include them or make the table just link to said places.